ui/app/mainui/AppMain.qml

```diff
@@ -2115,8 +2115,13 @@ Item {
                             }
                             totalTokensCount: appMain.tokensStore.plainTokensBySymbolModel.ModelCount.count
                             loading: false
-                            formatCurrencyAmount: function(amount) {
-                                return appMain.currencyStore.formatCurrencyAmount(amount, appMain.currencyStore.currentCurrency)
+                            currencySymbol: {
```

Comment:
But is it the same symbol we get from the "market" backend? If I select "EUR" in our Settings, will the backend return the correct values (converted from the what I assume is the default there, USD)?

Reply:
yes thats whats expected and works. updating currency in settings, leads to reloading wallet values in the selected currency.

Reply:
Cool!